code/datums/traits/neutral.dm

```diff
@@ -97,3 +97,21 @@
 
 /datum/quirk/monochromatic/remove()
 	quirk_holder.remove_client_colour(/datum/client_colour/monochrome)
+
+//MonkeStation Edit Start - Bald Quirk
+/datum/quirk/bald
+	name = "Bald"
+	desc = "Your hair seems to have gone missing. Luckily, you will spawn with a wig."
+	value = 0
+
+/datum/quirk/bald/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/obj/item/clothing/head/wig/W = new /obj/item/clothing/head/wig
+	W.hair_color = "#[H.hair_color]"
+	W.hair_style = H.hair_style
```

Comment:
Maybe color and style should be randomized, so species without hair options get something unique every round.
Non-humans can't access the hair menu, so they can't inherit anything they set.

Reply:
You can, you have to set them by picking a species with hair, selecting the hair options you'd like, then switching to the species you'd want to play. This spawns you with the hair you had selected. Regardless, I'd like to see a random option; since the wig allows you to change style itself, how about a random option in the wig's menu instead?